### DISCUSSION TITLE
Fedora package

### DISCUSSION THREAD

**Discussion by 0xMRTT** [mjakeman/text-engine#18] [General]
I've created a rpm package for text-engine 

https://copr.fedorainfracloud.org/coprs/0xmrtt/extension-manager/

https://github.com/0xMRTT/extension-manager-rpm/blob/main/text-engine/text-engine.spec

**Comment by mjakeman** [comment]
This is really cool, thanks for sharing!

I'll convert this to a discussion so we can leave it up.
